utils/cli/src/cmd.rs

```diff
@@ -0,0 +1,79 @@
+use anyhow::Result;
+use serde::Serialize;
+
+pub enum CommandOutput {
+    NoOutput,
+    Object(serde_json::Value),
+    Table {
+        columns: Vec<String>,
+        values: Vec<serde_json::Value>,
+        summary: Vec<serde_json::Value>,
+        header: Option<String>,
+    },
+    FormattedObject(Box<dyn FormattedObject>),
+}
+
+impl CommandOutput {
+    pub fn object<T: Serialize>(value: T) -> Result<Self> {
+        Ok(CommandOutput::Object(serde_json::to_value(value)?))
+    }
+
+    pub fn print(&self, json_output: bool) {
```

Comment:
Done, but it does not look great because `print_*` functions required defining params which originally were just taken directly from enum variant fields.

Reply:
Refactored it again. Checked some commands if I did not broke anything, but here is the problem - refactors should by done on tested code. I have no idea if the cmdline output is tested.